index.html

```diff
@@ -172,46 +177,53 @@ <h1>Tonic</h1>
-                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluM.png"
-                                    alt="not found"></a></li>
-                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluGit.png"
-                                    alt="not found"></a></li>
-                        <li><a href=""><img width="19.99px" height="19.96px" src="./assets/icons/BluVector.png"
-                                    alt="not found"></a></li>
+                <div class="about">
+                    <h1>About Myself</h1>
+                    <p>
+                        Hello I’m a software developer! I can help you build a product , feature or website Look through
+                        some of my work and experience! If you like what you see and have a project you need coded,
+                        don’t
+                        hestiate to contact me.
+                    </p>
+                    <a href="#" class="lets">LET'S CONNECT</a>
+                    <div class="contacts">
+                        <ul>
+                            <li><a href=""><img width="20px" height="16px" src="./assets/icons/bluTwitter.png"
+                                        alt="not found"></a></li>
+                            <li><a href=""><img width="20px" height="16px" src="./assets/icons/Linkedin-icon.png"
+                                        alt="not found"></a></li>
+                            <li><a href=""><img width="20px" height="16px" src="./assets/icons/BluM.png"
+                                        alt="not found"></a></li>
+                            <li><a href=""><img width="20px" height="16px" src="./assets/icons/BluGit.png"
+                                        alt="not found"></a></li>
+                            <li><a href=""><img width="20px" height="16px" src="./assets/icons/BluVector.png"
+                                        alt="not found"></a></li>
+                        </ul>
+                        <p><button>Get my resume</button></p>
+                    </div>
+                </div>
+                <div class="lang">
+                    <h3>Lanuages</h3>
+                    <ul class="languages">
+                        <li>
+                            <a href="#"><i class="fa-brands fa-html5"></i>html</a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa-brands fa-css3-alt"></i>css</a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa-brands fa-js"></i>javascript</a>
+                        </li>
                     </ul>
-                    <p><button>Get my resume</button></p>
+                    <button>frameworks <span class="fa fa-chevron-right"></span></button>
+                    <button>Skills <span class="fa fa-chevron-right"></span></button>
```

Comment:
- [ ] Great effort working on the desktop version of your portfolio project, In the about me section of the project I would kindly advise you to add the icon in front of the language to it as you have added it to the framework and skills. Also kindly make the framework and skills text to be as bold as the language, with this you have shown a kind of consistency in your design and as well follow the Figma design as much as possible.

Your Design 👇 

![elmas](https://user-images.githubusercontent.com/78634841/220855565-d9bfcb6d-72bd-4a43-a72e-98e7598003e9.png)